src/main/java/io/debezium/connector/yugabytedb/YugabyteDBStreamingChangeEventSource.java

```diff
@@ -116,19 +115,30 @@ public YugabyteDBStreamingChangeEventSource(YugabyteDBConnectorConfig connectorC
 
     @Override
     public void execute(ChangeEventSourceContext context, YugabyteDBPartition partition, YugabyteDBOffsetContext offsetContext) {
-        if (!snapshotter.shouldStream()) {
-            LOGGER.info("Streaming is not enabled in correct configuration");
-            return;
-        }
-
         // replication slot could exist at the time of starting Debezium so
         // we will stream from the position in the slot
         // instead of the last position in the database
-        boolean hasStartLsnStoredInContext = offsetContext != null;
+        // Get all partitions
+        // Get
+        Set<YBPartition> partitions = new YugabyteDBPartition.Provider(connectorConfig).getPartitions();
+        boolean hasStartLsnStoredInContext = offsetContext != null && !offsetContext.getTabletSourceInfo().isEmpty();
 
+        // LOGGER.info("SKSK The offset context is " + offsetContext + " partition is " + partition);
```

Comment:
This needs to be removed.

I guess we should also remove other stale comments like this which we may not need later to avoid confusion.